README.md

```diff
@@ -1,3 +1,5 @@
+# ⚠️ This plugin is no longer supported
```

Comment:
I think we should include the link to meta about this deprecation under this header as well.